tools/generate_hotkey.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+from zulipterminal.config.keys import HELP_CATEGORIES, KEY_BINDINGS
+
+
+general = []
+msg_actions = []
+msg_compose = []
+navigation = []
+searching = []
+stream_list = []
+
+CATEGORY = {
+    'general': general,
+    'msg_actions': msg_actions,
+    'msg_compose': msg_compose,
+    'navigation': navigation,
+    'searching': searching,
+    'stream_list': stream_list,
+}
+
+for key, item in KEY_BINDINGS.items():
+    CATEGORY[item['key_category']].append((item['help_text'], item['keys']))
+
+OUTPUT_FILE = "docs/hotkey.md"
+mdFile = open(OUTPUT_FILE, "w")
```

Comment:
Thank you for the pr, but it would be better if you use the `with open` syntax to do operations on the file. This would allow all closing and garbage collection to take place automatically when you move out. Else, you would have to handle it manually.

Reply:
Thank you for mentioning that. I'll make the change 👍🏼